ruby_programming/git/lesson_using_git_in_the_real_world.md

```diff
@@ -88,3 +88,14 @@ Sometimes (okay, maybe a lot of times) when you're working with Git, something g
 * [Oh sh!t git](http://ohshitgit.com/) is a quick reference to get you out of common Git problems.
 * This article on [How to undo (almost) anything with Git](https://github.blog/2015-06-08-how-to-undo-almost-anything-with-git/) will walk you through some of many options Git provides for undoing various mistakes.
 * If the problem you're facing is more advanced, you can click through [this more in-depth guide](https://sethrobertson.github.io/GitFixUm/fixup.html) to find the answer to your specific question.
+
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
+
+- <a class='knowledge-check-link' href='#commit-often'>How often should you commit?</a>
+- <a class='knowledge-check-link' href='#small-commit'>How large should your commits be?</a>
+- <a class='knowledge-check-link' href='#commit-message'>What should your commit message describe?</a>
+- <a class='knowledge-check-link' href='https://www.git-tower.com/blog/version-control-best-practices/'>Can you commit unfinished feature?</a>
+- <a class='knowledge-check-link' href='#send-changes'>Can you directly send your changes to the repository that you don't have access to?</a>
```

Comment:
```suggestion
- <a class='knowledge-check-link' href='#send-changes'>Can you directly send your changes to a repository that you don't own/ have write access to?</a>
```

Reply:
Looking good! Is 'own/' intentional or just a typo?

Reply:
it was intentional, my thought was that people may be more familiar with the concept of working on a repo that they own (have made on their own GitHub account) than repos that they've been granted write permission to since that doesn't usually happen until they are on the job or working with a team in some other way.

That space between the `/` and `write` shouldn't be there though, not sure why I hit space there.

If you disagree with my assessment and think that my suggestion is more confusing I'm open to discussing the point.

Reply:
I see. I initially didn't understand the phrase as " you don't own and have write access to" but as "you don't own have write access to" So I thought the sentence didn't make sense. But now I understand, yeah it sounds reasonable. 

People should be more familiar with the concept of owning a repository.